tools/sigclient/pkg/alerts/alerts_loadtest.go

```diff
@@ -77,6 +78,83 @@ func setUpLoggingToFileAndStdOut() {
 	// Set the output of the logger to the multi-writer
 	log.SetOutput(multiWriter)
 
+	return logFile
+}
+
+// tunnel the logs into SigLens
+func startVector(host string, logFile string) (*exec.Cmd, chan error, error) {
+	os.Setenv("LOG_FILE_PATH", logFile)
+	os.Setenv("HOST", host)
+
+	dataDir := filepath.Join("./vector_log_lib/data")
```

Comment:
Where is this directory coming from?

Reply:
So that is the directory, that vector uses internally to keep track of till where it has read the contents of a file or some other metadata information about a source. 

And I am just creating the folder in the root directory of SigClient. It will be created here:

<img width="342" alt="image" src="https://github.com/siglens/siglens/assets/29257812/d9ed1700-d1f8-4ccb-8838-7dc1db869906">